### DISCUSSION TITLE
update the Live Editor to 10.0

### DISCUSSION THREAD

**Discussion by meizy** [mermaid-js/mermaid#4123] [General]
It seems that the Live Editor is still on 9.3, so new stuff (like timeline) is not supported.
is there a reason for this? 

thanks,
Moshe

**Comment by Yokozuna59** [comment]
It has been update already in https://github.com/mermaid-js/mermaid-live-editor/blob/8edce6a3c1a0a86c5909e1ac3edb31dd989d349c/package.json#L76.

So I'll resolve the discussion.

**Reply by meizy** [reply]
Thanks
